tests/waku_rln_relay/test_wakunode_rln_relay.nim

```diff
@@ -469,25 +469,25 @@ procSuite "WakuNode - RLN relay":
           completionFut2.complete(true)
         if msg == wm3:
           completionFut3.complete(true)
-    
+
     # mount the relay handler for node2
     node2.subscribe((kind: PubsubSub, topic: DefaultPubsubTopic), some(relayHandler))
     await sleepAsync(2000.millis)
 
     await node1.publish(some(DefaultPubsubTopic), wm1)
-    await sleepAsync(10.seconds)
     await node1.publish(some(DefaultPubsubTopic), wm2)
```

Comment:
@rymnc Mind clarifying why this sleep was needed? Epoch is hardcoded above, so perhaps not needed?

Reply:
it was for message propagation i believe, due to some race condition we had earlier